app/views/fields/carrierwave/_show.html.erb

```diff
@@ -22,7 +22,7 @@ for this attribute
     <span>No file(s) uploaded yet</span>
   <% end %>
 <% else %>
-  <% if field.image.present? %>
+  <% if field.image.present? && !field.data.model.new_record? %>
```

Comment:
Can you please amend the second half of this to `field.data.model.persisted?`?

I think positive conditions reads better than negatives!